pkg/scaffold/v2/main.go

```diff
@@ -155,14 +155,27 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+
+    // Add flags for prometheus metrics
+    flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+
+    // Add flags for leader election
+    flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.Parse()
 
-	ctrl.SetLogger(zap.Logger(true))
+    // Add lib for logs 
+    ctrl.SetLogger(zap.Logger(true))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    // Get a config to talk to the apiserver
+    cfg, err := config.GetConfig()
+    if err != nil {
+       setupLog.Error(err, "unable to get config manager")
```

Comment:
this error message is incorrect.  I should be something like "unable to get configuration to talk to the api server" or something